dvc/utils/__init__.py

```diff
@@ -80,6 +99,10 @@ def file_md5(fname, tree=None):
             with open_func(fname, "rb") as fobj:
                 _fobj_md5(fobj, hash_md5, binary, pbar.update)
 
+        if filtered is not None:
+            from dvc.utils.fs import remove
+
+            remove(filtered)
```

Comment:
not sure if this is required - maybe automatically handled elsewhere (i.e. entire tmpdir deleted before exit)